src/main/java/com/epam/reportportal/extension/azure/command/connection/TestConnectionCommand.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.reportportal.extension.azure.command.connection;
+
+import com.epam.reportportal.extension.PluginCommand;
+import com.epam.reportportal.extension.azure.rest.client.ApiClient;
+import com.epam.reportportal.extension.azure.rest.client.ApiException;
+import com.epam.reportportal.extension.azure.rest.client.ApiResponse;
+import com.epam.reportportal.extension.azure.rest.client.Configuration;
+import com.epam.reportportal.extension.azure.rest.client.api.ProjectsApi;
+import com.epam.reportportal.extension.azure.rest.client.auth.HttpBasicAuth;
+import com.epam.reportportal.extension.azure.rest.client.model.TeamProject;
+import com.epam.ta.reportportal.entity.integration.Integration;
+import com.epam.ta.reportportal.exception.ReportPortalException;
+import com.epam.ta.reportportal.ws.model.ErrorType;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestConnectionCommand implements PluginCommand<Boolean> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestConnectionCommand.class);
+    private static final String API_VERSION = "6.0";
+
+    @Override
+    public Boolean executeCommand(Integration integration, Map<String, Object> params) {
+
+        ApiClient defaultClient = Configuration.getDefaultApiClient();
+
+        String organizationUrl = params.get("url").toString();
+        String organizationName = organizationUrl.replace(defaultClient.getBasePath(), "");
+        String projectName = params.get("project").toString();
+        String personalAccessToken = params.get("oauthAccessKey").toString();
+
+        HttpBasicAuth basicAuth = (HttpBasicAuth) defaultClient.getAuthentication("accessToken");
+        basicAuth.setPassword(personalAccessToken);
+
+        ProjectsApi projectsApi = new ProjectsApi(defaultClient);
+
+        try {
+            ApiResponse<TeamProject> response = projectsApi.projectsGetWithHttpInfo(organizationName, projectName,
+                    API_VERSION, false, false);
+            return response.getStatusCode() == 200;
```

Comment:
if status code is not 200 then it would be better to read response and throw an error.
like "Unable to connect to Azure DevOps: response code is {response.getStatusCode()}, message is " + response.(get response body)()

Reply:
If status code is not 200, an ApiException will be thrown and the catch block will be triggered